pysb/bng.py

```diff
@@ -49,14 +52,25 @@ class BngBaseInterface(object):
 
     @abc.abstractmethod
     def __init__(self, model=None, verbose=False, cleanup=False,
-                 output_prefix=None, output_dir=None):
+                 output_prefix=None, output_dir=None,
+                 model_additional_species=None,
+                 model_population_maps=None):
+        self._logger = get_logger(__name__,
+                                  model=model,
+                                  log_level=verbose)
+        if model:
```

Comment:
I mean I get the model present/absent part, but why is it switching on accessing `self._logger.logger` vs. `self._logger`? What even *is* `self._logger.logger`?